ts/test/local/conductor.ts

```diff
@@ -300,6 +306,80 @@ test("Local Conductor - install app with deferred memproofs", async (t) => {
   await cleanAllConductors();
 });
 
+test("Local Conductor - install app with roles settings", async (t) => {
+  const { servicesProcess, signalingServerUrl } = await runLocalServices();
+  const conductor = await createConductor(signalingServerUrl);
+
+  const originTime = Date.now();
+  const quantumTime: Duration = {
+    secs: originTime,
+    nanos: 0,
+  };
+
+  const progenitorKey = Uint8Array.from(await fakeAgentPubKey());
+
+  const app = await conductor.installApp(
+    {
+      bundle: {
+        manifest: {
+          manifest_version: "1",
+          name: "app",
+          roles: [
+            {
+              name: ROLE_NAME,
+              provisioning: {
+                strategy: CellProvisioningStrategy.Create,
+                deferred: false,
+              },
+              dna: {
+                path: realpathSync(FIXTURE_DNA_URL),
+                modifiers: { network_seed: "some_seed" },
+              },
+            },
+          ],
+          membrane_proofs_deferred: true,
+        },
+        resources: {},
+      },
+    },
+    {
+      rolesSettings: {
+        [ROLE_NAME]: {
+          type: "Provisioned",
+          membrane_proof: new Uint8Array(6),
+          modifiers: {
+            network_seed: "hello",
+            properties: yaml.dump({ progenitor: progenitorKey }),
+            origin_time: originTime,
+            quantum_time: quantumTime,
+          },
+        },
+      },
+    }
+  );
+
+  const port = await conductor.attachAppInterface();
+  const issued = await conductor
+    .adminWs()
+    .issueAppAuthenticationToken({ installed_app_id: app.installed_app_id });
+  const appWs = await conductor.connectAppWs(issued.token, port);
+
+  const appInfo = await appWs.appInfo();
+  const provisionedCell: ProvisionedCell =
+    appInfo.cell_info[ROLE_NAME][0][CellType.Provisioned];
+  t.equal(provisionedCell.dna_modifiers.network_seed, "hello");
+  t.deepEqual(
+    yaml.load(decode(provisionedCell.dna_modifiers.properties) as string),
+    { progenitor: progenitorKey }
+  );
+  t.equal(provisionedCell.dna_modifiers.origin_time, originTime);
+  t.deepEqual(provisionedCell.dna_modifiers.quantum_time, quantumTime);
```

Comment:
not sure if its overkill but should we add an assertion that the membrane proof is commited to our chain?